src/main/java/seedu/task/logic/commands/EditCommand.java

```diff
@@ -87,19 +99,26 @@ private static Task createEditedTask(ReadOnlyTask taskToEdit,
     public static class EditTaskDescriptor {
         private Optional<Description> description = Optional.empty();
         private Optional<UniqueTagList> tags = Optional.empty();
+        private Optional<String> durationStart = Optional.empty();
+        private Optional<String> durationEnd = Optional.empty();
```

Comment:
Why not just use a `Duration` rather than storing 2 strings here? It would clean up the getters/setters below as well